cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -6627,4 +6627,12 @@
     </description>
   </property>
 
+  <property>
+    <name>ui.default.poll.interval.millis</name>
+    <value>10000</value>
+    <description>
+      The default polling interval in milliseconds for poll calls from cdap-ui.
+    </description>
+  </property>
+
```

Comment:
can't the default value be maintained somewhere in cdap-ui? 

https://github.com/cdapio/cdap-ui/blob/ef1f2aec9ab9777b25ae8953d32b8354695dc7aa/server/config/development/cdap.json#L2

It is always better if it can be present close to the context.